web/packages/api/src/xcmBuilder.ts

```diff
@@ -2,13 +2,21 @@ import { Registry } from "@polkadot/types/types"
 import { beneficiaryMultiAddress } from "./utils"
 import { ETHER_TOKEN_ADDRESS } from "./assets_v2"
 
-export const HERE_LOCATION = { parents: 0, interior: "Here"  }
-export const DOT_LOCATION = { parents: 1, interior: "Here"  }
+export const HERE_LOCATION = { parents: 0, interior: "Here" }
+export const DOT_LOCATION = { parents: 1, interior: "Here" }
 
 const ethereumNetwork = (ethChainId: number) => ({
     GlobalConsensus: { Ethereum: { chain_id: ethChainId } },
 })
 
+export const polkadotNetwork = () => ({
```

Comment:
Just use constants instead of lambda.

Reply:
Addressed in https://github.com/Snowfork/snowbridge/pull/1463/commits/f309c26ea81cff0207f542a9ba8cfecc82e561de.